samples/dotnet/clientcreds/readme.md

```diff
@@ -2,5 +2,52 @@
 
 Sample implementation of the Lenus Health platform demonstrating use client_credentials grant type flow to support requesting agency over a patient by a clinician within an organisation.
 
+## Running locally
+
+Open solution in IDE of choice and run the Lenus.Samples.ClientCredentialsFlow project. If using Visual Studio you will be prompted to install a self-signed certificate on your machine for https redirection. Running via CLI or another IDE (VS code) will not do this step automatically so it will be necessary to install the certificates yourself. This can be done by following these commands:
+
+```
+dotnet dev-certs https --clean
+dotnet dev-certs https --trust
+```
+
+Run `dotnet dev-certs https --trust` on macOS and Windows or alternatively trust the dev certificate via your browser
+
 ## Required Setup
 
+You will need to set up a client via identity admin configured to use the client_credentials grant type. This can be done via a deployed environment such as test or in a locally running instance.
```

Comment:
These samples are mostly targeted at third party developers, so best to stick to 'sandbox' and not mention test or local alternatives.